src/pages/instances/InstanceSearchFilter.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { FC } from "react";
+import { LxdInstance, LxdInstanceStatus } from "types/instance";
+import {
+  InstanceFilters,
+  instanceStatuses,
+  instanceTypes,
+} from "util/instanceFilter";
+import { SearchAndFilter } from "@canonical/react-components";
+import {
+  SearchAndFilterData,
+  SearchAndFilterChip,
+} from "@canonical/react-components/dist/components/SearchAndFilter/types";
+import { InstanceListType } from "util/instanceOptions";
+import isEqual from "lodash/isEqual";
+
+interface Props {
+  instances: LxdInstance[];
+  filters: InstanceFilters;
+  setFilters: (newFilters: InstanceFilters) => void;
+}
+
+const InstanceSearchFilter: FC<Props> = ({
+  instances,
+  filters,
+  setFilters,
+}) => {
+  const profileSet = [
+    ...new Set(instances.flatMap((instance) => instance.profiles)),
+  ];
+
+  const searchAndFilterData: SearchAndFilterData[] = [
+    {
+      id: 1,
+      heading: "Status",
+      chips: instanceStatuses.map((status) => {
+        return { lead: "status", value: status };
+      }),
+    },
+    {
+      id: 2,
+      heading: "Instance type",
+      chips: instanceTypes.map((type) => {
+        return { lead: "type", value: type.uiName };
+      }),
+    },
+    {
+      id: 3,
+      heading: "Profile",
+      chips: profileSet.map((profile) => {
+        return { lead: "profile", value: profile };
+      }),
+    },
+  ];
+
+  const onSearchDataChange = (searchData: SearchAndFilterChip[]) => {
+    const newQueries = searchData
+      .filter((chip) => chip.quoteValue)
+      .map((chip) => chip.value);
```

Comment:
We can do the `chip.value.toLowerCase()` here, so we don't do it for every instance that is filtered.